docroot/sites/example.settings.local.php

```diff
@@ -130,6 +130,26 @@
  */
 $settings['skip_permissions_hardening'] = TRUE;
 
-error_reporting(E_ALL);
-ini_set('display_errors', TRUE);
-ini_set('display_startup_errors', TRUE);
```

Comment:
Probably would be nice to keep this in here when we get the scaffolding back up and running. This will help us catch bugs early/often.

Reply:
So these things somehow got removed automatically during some of the install/init process. Not sure which thing did this. But I can undo it.